doc/tutorials/acrn_configurator_tool.rst

```diff
@@ -73,7 +79,8 @@ such as a board configuration file, scenario configuration file, and launch
 scripts. If it finds any, it will delete them.
```

Comment:
Please add borders to images in the final version.  This is mostly an issue with images that are white where it is difficult to visually see where the image ends.

Reply:
The published version has a border. https://projectacrn.github.io/latest/tutorials/acrn_configurator_tool.html

Reply:
If you're looking at the github rendering it won't have a drop shadow (as amy mentioned).  I added an image class that we can use to add a drop shadow when the picture would benefit (such a screen shot with a white background):
```
 .. image:: images/configurator-newscenario01.png
          :align: center
          :class: drop-shadow
```